src/components/settings/sync-history.tsx

```
@@ -0,0 +1,253 @@
+
+export function SyncHistory() {
+  const [syncJobs, setSyncJobs] = useState<SyncJob[]>([]);
+  const [loading, setLoading] = useState(true);
+  const [page, setPage] = useState(1);
+  const [hasMore, setHasMore] = useState(false);
+
+  const ITEMS_PER_PAGE = 10;
+
+  useEffect(() => {
+    loadSyncHistory();
+  }, [page, loadSyncHistory]);
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)